lib/rex/post/meterpreter/ui/console.rb

```diff
@@ -122,7 +122,7 @@ def run_command(dispatcher, method, arguments)
   def log_error(msg)
     print_error(msg)
 
-    elog(msg, 'meterpreter')
+    elog(msg, src='meterpreter')
```

Comment:
As we caught up on a call - this is assigning variables unintentionally, shouldn't need to do this 👀